include/libopencm3/stm32/l0/timer.h

```diff
@@ -138,31 +138,53 @@ LGPL License Terms @ref lgpl_license
 #define LPTIM_IER_UPIE			(1 << 5)
 #define LPTIM_IER_DOWNIE		(1 << 6)
 
+/* --- LPTIM_CFGR values ----------------------------------------------------- */
+
+/* CKSEL: Select internal (0) or external clock source (1) */
 #define LPTIM_CFGR_CKSEL		(1 << 0)
 
 #define LPTIM_CFGR_CKPOL_SHIFT		1
+#define LPTIM_CFGR_CKPOL_MASK		0x03
 #define LPTIM_CFGR_CKPOL		(3 << LPTIM_CFGR_CKPOL_SHIFT)
+/** @defgroup lptim_cfgr_ckpol LPTIM_CFGR CKPOL Clock Polarity
+* @ingroup timer_defines
```

Comment:
You can skip the @ingroup when it's naturally inside that group.  You only need to explicitly do an @ingroup if you want to put it "elsewhere"

Reply:
hmm, that does not seem to work here without explicit ingroup :( - not sure how/why right now, but same thing is done in timer_common_all.h. I'll have a look.